Assets/Scripts/Drinkables/BlueBuffaloBeer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+
+namespace Drinkables
+{
+    public class BlueBuffaloBeer : AbstractDrinkable
```

Comment:
Vignette effect not working on this beer

Reply:
Fixed